styles.css

```diff
@@ -0,0 +1,25 @@
+body{
+    font-family: 'Kaushan Script', cursive;
+}
+h1{
+    font-size: 60px;
+}
+h2{
+    font-size: 30px;
+}
+p{
+    font-size: 18px;
+}
+h1{
+    font-weight: bold;
+}
+h1{
+    text-decoration: underline;
+}
```

Comment:
Same note about combining your rulesets. It's easier to keep this code tidy and know what it's doing if there's only one instance of an `h1` selector, so you want to combine this code with the code on lines 4-6, like so:

```css
h1 {
    font-size: 60px;
    text-decoration: underline;
}
```